Leibit.Core/Data/sued/RSI.xml

```diff
@@ -0,0 +1,330 @@
+      <alternative track="41"/>
+      <alternative track="45"/>
+      <alternative track="46"/>
+    </track>
+    <track name="45">
+      <block name="56G45"/>
+      <alternative track="42"/>
+      <alternative track="43"/>
+      <alternative track="44"/>
+      <alternative track="41"/>
+      <alternative track="46"/>
+    </track>
+    <track name="46">
+      <block name="56G46"/>
+      <alternative track="42"/>
+      <alternative track="43"/>
+      <alternative track="44"/>
+      <alternative track="45"/>
+      <alternative track="41"/>
+    </track>
+  </station>
+
+  <station name="Böhringen-Rickelshausen" short="RBER" refNr="56" scheduleFile="böhri___.abf">
+    <track name="2" calculateDelay="false">
+      <block name="56B1"/>
+    </track>
+    <track name="1" calculateDelay="false">
+      <block name="56BP"/>
+    </track>
+  </station>
+
+  <station name="Singen (Hohentwiel) Industriegebiet" short="RSII" refNr="55" scheduleFile="b55_____.abf">
+    <track name="2" calculateDelay="false">
+      <block name="55BN1"/>
+    </track>
+    <track name="1" calculateDelay="false">
+      <block name="55B2"/>
+    </track>
+  </station>
+  
+  <station name="Singen (Hohentwiel)" short="RSI" refNr="55" scheduleFile="g55_____.abf" localOrderFile="bf55____.abf">
+    <track name="1">
+      <block name="55G1"/>
+    </track>
+    <track name="51">
+      <block name="55G1"/>
+    </track>
+    <track name="2">
+      <block name="55G2"/>
+    </track>
+    <track name="3">
+      <block name="55G3"/>
+    </track>
+    <track name="4">
+      <block name="55G4"/>
+    </track>
+    <track name="5">
+      <block name="55G5"/>
+    </track>
+    <track name="6">
+      <block name="55G6"/>
+    </track>
+    <track name="8">
+      <block name="55G8"/>
+    </track>
+    <track name="9">
+      <block name="55G9"/>
+    </track>
+    <track name="10">
+      <block name="55G10"/>
+    </track>
+    <track name="11">
+      <block name="55G11"/>
+    </track>
+    <track name="12">
+      <block name="55G12"/>
+    </track>
+    <track name="13">
+      <block name="55G13"/>
+    </track>
+    <track name="14">
+      <block name="55G14"/>
+    </track>
+    <track name="15">
+      <block name="55G15"/>
+    </track>
+    <track name="116">
+      <block name="55G116"/>
+    </track>
+  </station>
+
+  <station name="Gottmadingen" short="RG" refNr="54" scheduleFile="g54_____.abf" localOrderFile="bf54____.abf">
+    <track name="1">
+      <block name="54G1"/>
+    </track>
+    <track name="2">
+      <block name="54G2"/>
+    </track>
+  </station>
+
+  <station name="Bietingen" short="RBGN" refNr="54" scheduleFile="b54_____.abf">
+    <track name="1" calculateDelay="false">
+      <block name="53BN"/>
+    </track>
+    <track name="2" calculateDelay="false">
+      <block name="54BP"/>
+    </track>
+  </station>
+
+  <station name="Thayngen" short="RTG" refNr="53" scheduleFile="g53_____.abf" localOrderFile="bf53____.abf">
```

Comment:
Hier fehlen mir die Gleise 51 und 52. Wenn diese nicht im Fahrplan (in der .abf Datei) auftauchen, ist das okay. Falls doch, müssen die beiden Gleise hier noch ergänzt werden.

Reply:
In der Demo kommen die Gleise ja nicht vor. Dann lassen wir sie erstmal weg. Man kann sie ja immer noch ergänzen.